library/core/src/mem/maybe_uninit.rs

```diff
@@ -456,8 +454,29 @@ impl<T> MaybeUninit<T> {
     /// // x is initialized now:
     /// let s = unsafe { x.assume_init() };
     /// ```
-    #[unstable(feature = "maybe_uninit_extra", issue = "63567")]
-    #[rustc_const_unstable(feature = "maybe_uninit_extra", issue = "63567")]
+    ///
+    /// Usage of this method that avoids an unsafe:
```

Comment:
I might write a little more here:

> This method can be used to avoid `unsafe` in some cases. This example shows a part of an implementation of a fixed sized arena that lends out pinned references. With `write`, we can avoid the need to write through a raw pointer: